src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -202,11 +214,17 @@ static CliRootCommand CreateCommandLineParser()
             SessionId: sessionId,
             SharedDependenciesPath: sharedDependenciesPath,
             ExtensionAssemblyPaths: extensionAssemblyPaths,
-            ExtensionLogDirectory: extensionLogDirectory);
+            ExtensionLogDirectory: extensionLogDirectory,
+            PipeName: pipe);
 
         return RunAsync(serverConfiguration, cancellationToken);
     });
 
     return rootCommand;
 }
 
+static string GetDotNetPipeConnectionString(string pipeName)
+{
+    return "\\\\.\\" + pipeName;
+}
```

Comment:
Does this syntax also work on Mac/Linux pipes? Or should we expect that the host is providing the Win32 name?

Reply:
I am not sure! Attempting to test on mac now